libraries/lib-ffmpeg-support/impl/avcodec/AVCodecContextWrapperImpl.inl

```diff
@@ -27,12 +27,12 @@ struct Converters<int16_t>
 
    static int16_t Convert(int32_t value)
    {
-      return static_cast<int16_t>(value >> 1);
+      return static_cast<int16_t>(value >> 16);
```

Comment:
Now that I've started thinking of it - both variants are incorrect. Right-shift is UB in C++ :( 

Yours is more correct, though. I will rewrite it using a similar approach to our regular sample format conversions, without dithering (this can be fixed later!)

Reply:
https://en.cppreference.com/w/cpp/language/operator_arithmetic

Strictly speaking, right shift of signed and negative values has “implementation defined” behavior, which is not the much worse category of UB.

Reply:
It was late night when I wrote these code changes.  Something needed doing here but I didn’t get it all right.

Reply:
Yeah, same for me when I was writing the original code :-)

Thank you a lot for pointing it out, this was a really stupid error, not exposed in my testing